scratch/fetch_info.py

```
@@ -0,0 +1,91 @@
+detailed_items = []
+seen_categories = set()
+
+for item in candidates:
+    asin = item["asin"]
+    cat = item["category"]
+
+    if cat in seen_categories:
+        continue
+
+    print(f"Fetching details for ASIN {asin} ({cat})...")
+    subprocess.run(["uv", "run", "python", "scripts/creators_get_item.py", asin])
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)